### PR TITLE
explain usage (running vs. occupied)

### PR DESCRIPTION
**Pull request by matthiasbernhardt**

### CODE REVIEW

File: user/pages/04.Reference/03.compute/docs.en.md [hunk 1]
```diff
@@ -240,3 +240,12 @@ This security group can now be connected to a port of your network:
...
 
 The security group "default" is added in this example, since this group is taking care of allowing outbound traffic.
+
+
+### Are stopped instances counted regarding quota or ondemand billing?
+
+Any occupied resources are considered "used" and counted against your [quota limits](../../05.Background/03.resource-limits/docs.en.md#usage). They will also be counted against your upfront commitment or charged on demand.
+When you create an instance, volume, snapshot, object, loadbalancer, floating ip, dns zone, rescources will be allocated and so they will be counted and charged, no matter if they are actually running, attached, assigned.
+To save money or free quota, it is thus not sufficient to stop, detach or unassign them, you must delete them completely to free the resources so that we can reuse them.
+It is part of higher level automation like kuberenetes or terraform or similar tools to recreate them if needed.
```
Comment (lgrebe-sys) [inline user/pages/04.Reference/03.compute/docs.en.md:250]: Just a typo for "rescources" and "kubernetes"! Rest lgtm

File: user/pages/05.Background/03.resource-limits/docs.en.md [hunk 2]
```diff
@@ -39,5 +39,10 @@ Some customers prefer to commit to a certain volume paid upfront, other prefer p
 
 While quota limits apply per project and region, your upfront commitment can be applied over all regions. This comes in handy, if you have a not fully redundant, but fully automated infrastructure in one of our regions and we announce a maintenance. You can get the same quota limits in another region, build a copy of your infrastructure, switch over and tear down the now unused infrastructure in the old region. While the baseline of you setup is covered by the commitment, you only have to pay the exceeding resources on demand for the short duration where the setup is doubled. The idle buffers do not need to be paid.
 
+### Usage
 
+Any occupied resources are considered "used" and counted against your quota limits. They will also be counted against your upfront commitment or charged on demand.
+When you create an instance, volume, snapshot, object, loadbalancer, floating ip, dns zone, rescources will be allocated and so they will be counted and charged, no matter if they are actually running, attached, assigned.
```
Comment (lgrebe-sys) [inline user/pages/05.Background/03.resource-limits/docs.en.md:45]: Just a copy paste of typos for "rescources" and "kuberenetes" =)
Rest lgtm!

**Review by lgrebe-sys — CHANGES_REQUESTED** [review]
Just minor typos, rest looks good already =)

File: user/pages/05.Background/03.resource-limits/docs.en.md (hunk 2 above)
Comment (matthiasbernhardt) [reply user/pages/05.Background/03.resource-limits/docs.en.md]: 🙈

**Review by lgrebe-sys — APPROVED** [review]

**Pull request merged**